include/api.php

```diff
@@ -3776,8 +3777,8 @@ function api_friendica_notification($type) {
 
 		$a = get_app();
 
-		if (api_user()===false) throw new ForbiddenException();
-		if ($a->argc!==3) throw new BadRequestException("Invalid argument count");
+		if (api_user()===false) throw new HTTP\ForbiddenException();
```

Comment:
Standards: Please add braces to this conditional statement and spaces around the `===` operator.